rep-2000.rst

```diff
@@ -1304,6 +1201,66 @@ Build System Support:
 - cmake
 - setuptools
 
+Middleware Implementation Support:
+
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+----------+
+| Middleware Library       | Middleware Provider    | Support Level | Platforms                  | Architectures                  | Version  |
++==========================+========================+===============+============================+================================+==========+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS      | Tier 1        | All Platforms              | All Architectures              | 2.14.4   |
```

Comment:
Minor thought: Can we change "Version" to "Middleware Version" just so that people don't think it applies to the `rmw_*` implementation itself?

Reply:
Good idea.  I've taken this change.